ansible-lint.sh

```
@@ -36,24 +36,29 @@ initialization() {
   git fetch -q origin ${GITHUB_REF}
   git checkout -q FETCH_HEAD
   echo "Pull request sources downloaded succesfully"
-
+  if [ ! -f ansible-lint.conf ] ; then
+      echo "No ansible-lint available. Skipping linting."
+      exit 0
```

I think it would be better to return an error in this case. There should be a configuration file.